doc/dialects/postgresql.md

```diff
@@ -0,0 +1,194 @@
+# PostgreSQL SQL Dialect
+
+[PostgreSQL](https://www.postgresql.org/) is an open-source  Relational Database Management System (RDBMS).
+
+## Uploading the JDBC Driver to EXAOperation
+
+First download the [PostgreSQL JDBC driver](https://jdbc.postgresql.org/).
+Driver version 42.2.6 or later is recommended if you want to establish a TLS-secured connection. 
+
+1. [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm)
+1. Upload the driver to BucketFS
+
+## Installing the Adapter Script
+
+Upload the latest available release of [Virtual Schema JDBC Adapter](https://github.com/exasol/virtual-schemas/releases) to Bucket FS.
+
+Then create a schema to hold the adapter script.
+
+```sql
+CREATE SCHEMA ADAPTER;
+```
+
+The SQL statement below creates the adapter script, defines the Java class that serves as entry point and tells the UDF framework where to find the libraries (JAR files) for Virtual Schema and database driver.
+
+```sql
+CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
+  %scriptclass com.exasol.adapter.RequestDispatcher;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-5.0.4-bundle-4.0.3.jar;
```

Comment:
Name of the VS needs to be adapted. Otherwise this will lead to confusion with the bundle.

Reply:
right!

Reply:
how about now?